lib/mongoose_push.ex

```diff
@@ -72,7 +72,16 @@ defmodule MongoosePush do
 
     {time, push_result} =
       if pool == nil do
-        Logger.error(~s"No pool matching mode=#{mode} and tags=#{inspect(tags)}")
+        Logger.error(
+          fn ->
+            "choose_pool"
```

Comment:
This message should be "human readable", yet without any variables. Here I would suggest - "No matching pool found."